bitnami/wordpress/templates/ingress.yaml

```diff
@@ -46,18 +43,15 @@ spec:
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" $) "servicePort" "http" "context" $) | nindent 14 }}
     {{- end }}
-  {{- if or .Values.ingress.tls .Values.ingress.extraTls }}
+  {{- if or (and .Values.ingress.tls (or (include "wordpress.ingress.certManagerRequest" .Values.ingress.annotations) .Values.ingress.selfSigned)) .Values.ingress.extraTls }}
   tls:
-    {{- if .Values.ingress.tls }}
+    {{- if and .Values.ingress.tls (or (include "wordpress.ingress.certManagerRequest" .Values.ingress.annotations) .Values.ingress.selfSigned) }}
     - hosts:
         - {{ .Values.ingress.hostname | quote }}
-        {{- range .Values.ingress.extraHosts }}
-        - {{ .name | quote }}
-        {{- end }}
```

Comment:
@juan131 Hi, I noticed that this PR reverses the changes in https://github.com/bitnami/charts/pull/4953. Could you please explain why this is necessary?